packages/trip-form/src/components.story.js

```diff
@@ -5,10 +5,10 @@ import * as Icons from "@opentripplanner/icons";
 
 import * as Core from ".";
 
-import commonModes from "./__mocks__/modes";
-import modeOptions from "./__mocks__/mode-options";
-import submodeOptions from "./__mocks__/submode-options";
-import trimet from "./__mocks__/trimet.styled";
+import commonModes from "./test-utils/modes";
+import modeOptions from "./test-utils/mode-options";
+import submodeOptions from "./test-utils/submode-options";
+import trimet from "./test-utils/trimet-styled";
```

Comment:
Why all the file name changes? Just curious.

Reply:
I can't remember, maybe it seemed like they would be better organized in this folder since they were solely for helping render stories.